pkg/types/kustomization.go

```diff
@@ -136,3 +142,16 @@ type DataSources struct {
 	// i.e. a Docker .env file or a .ini file.
 	EnvSource string `json:"env,omitempty" yaml:"env,omitempty"`
 }
+
+// ImageTag contains an image and a new tag, which will replace the original tag
+type ImageTag struct {
+	// Name is the name for an image without tag
+	Name string `json:"name,omitempty" yaml:"name,omitempty"`
+
+	// Tag will replace the original tag associated with Name at runtime
+	Tag string `json:"tag,omitempty" yaml:"tag,omitempty"`
```

Comment:
suggest calling this NewTag

// NewTag is the value to use in replacing the original image tag.